packages/packages.csv

```diff
@@ -58,9 +59,6 @@ Microsoft.AspNetCore.HttpOverrides,ship
 Microsoft.AspNetCore.Identity,ship
 Microsoft.AspNetCore.Identity.EntityFrameworkCore,ship
 Microsoft.AspNetCore.IISPlatformHandler,noship
-Microsoft.AspNetCore.Internal.libuv-Darwin,noship
-Microsoft.AspNetCore.Internal.libuv-Linux,ship
-Microsoft.AspNetCore.Internal.libuv-Windows,noship
```

Comment:
We're going to completely skip the `libuv-build-windows` repo. If we want to preserve the internal packages in our build results, we should `libuv-package` to drop these packages in the artifacts directory.